Makefile

```diff
@@ -50,8 +63,19 @@ RUNSETUPDACMOCKL1 := $(DOCKERCOMPOSE) up -d $(DOCKERCOMPOSESETUPDACMOCKL1)
 RUNSETUPDACSEPOLIA := $(DOCKERCOMPOSE) up -d $(DOCKERCOMPOSESETUPDACSEPOLIA)
 RUNEXPLORERL1 := $(DOCKERCOMPOSE) up -d $(DOCKERCOMPOSEEXPLORERL1)
 RUNEXPLORERL1DB := $(DOCKERCOMPOSE) up -d $(DOCKERCOMPOSEEXPLORERL1DB)
+RUNEXPLORERFRONTENDL1 := $(DOCKERCOMPOSE) up -d $(DOCKERCOMPOSEEXPLORERFRONTENDL1)
+RUNEXPLORERSTATSDBL1 := $(DOCKERCOMPOSE) up -d $(DOCKERCOMPOSEEXPLORERSTATSDBL1)
+RUNEXPLORERSTATSL1 := $(DOCKERCOMPOSE) up -d $(DOCKERCOMPOSEEXPLORERSTATSL1)
+RUNEXPLORERSMARTCONTRACTVERIFIERL1 := $(DOCKERCOMPOSE) up -d $(DOCKERCOMPOSEEXPLORERSMARTCONTRACTVERIFIERL1)
+RUNEXPLORERVISUALIZERL1 := $(DOCKERCOMPOSE) up -d $(DOCKERCOMPOSEEXPLORERVISUALIZERL1)
+RUNEXPLORERSIGPROVIDERL1 := $(DOCKERCOMPOSE) up -d $(DOCKERCOMPOSEEXPLORERSIGPROVIDERL1)
```

Comment:
similar as above. the naming should not be L1

also applied to the STOPXXX commands

Reply:
Fixed